### PR TITLE
sanctum 1.7.0 (new cask)

### PR DESCRIPTION
**Pull request by daeho-ro**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/s/sanctum.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  url "https://sanctum.ai/darwin-#{folder}/Sanctum_#{version}_#{arch}.dmg",
+      user_agent: :fake
+  name "Sanctum"
+  desc "Run & interact with full-featured open-source LLMs locally"
+  homepage "https://sanctum.ai/"
+
+  livecheck do
+    url :homepage
+    regex(/Sanctum[._-]?v?(\d+(?:\.\d+)+)[._-]?\w+\.dmg/i)
+    strategy :page_match
```
Comment (khipp) [inline Casks/s/sanctum.rb:18]:
```suggestion
    url :homepage
    regex(/href=.*?Sanctum[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg/i)
```

We can omit specifying the `strategy`, as `PageMatch` is used by default here.
When matching file names, we typically restrict matching to `href` attributes and do not make delimiters optional (see [documentation](https://docs.brew.sh/Brew-Livecheck#file-names)).

File: Casks/s/sanctum.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  folder = on_arch_conditional arm: "aarch64", intel: "x86_64"
+
+  version "1.7.0"
+  sha256 arm:   "7d0826a6aa7b0e2c502e786eb00a9339b1d48871ad216958b87ac4cd68ee5a6c",
+         intel: "8e78706086b346326441f53206074eb2ab087fb0ef9a67f8c6a154451c14ce97"
+
+  url "https://sanctum.ai/darwin-#{folder}/Sanctum_#{version}_#{arch}.dmg",
+      user_agent: :fake
+  name "Sanctum"
+  desc "Run & interact with full-featured open-source LLMs locally"
```
Comment (khipp) [inline Casks/s/sanctum.rb:12]:
```suggestion
  desc "Run LLMs locally"
```

**Review by khipp — APPROVED** [review]
Thank you, @daeho-ro!

**Comment by khipp** [comment]
@daeho-ro This PR was accidentally closed as well.

**Comment by daeho-ro** [comment]
New PR is here, sorry for that.
- https://github.com/Homebrew/homebrew-cask/pull/187706

**Pull request closed without merging**